docs/pages/guides/notifications-services.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: Using Push Notifications
+description: Learn about different push notification services.
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+
+Expo apps can work with any notification service or capabilities offered by the operating system thanks to [config plugins](/config-plugins/introduction/). The following services provide purpose-built Expo integrations for implementing push notifications in your app:
```

Comment:
```suggestion
Expo apps can work with any notification service or capabilities offered by the operating system. The following services provide purpose-built Expo integrations for implementing push notifications in your app:
```

the role of config plugins is not directly visible to me here. Either remove or expand on the relation.